lua/cfc_notifications/shared/sh_presets.lua

```diff
@@ -23,26 +23,46 @@ CFCNotifications.registerNotificationType( "Buttons", function( CONTEXT )
     -- These require player input, so are more important than other notifications
     CONTEXT._priority = CFCNotifications.PRIORITY_NORMAL
 
-    local btnH = 30
+    local btnH = 45
     local btnBottomMargin = 10
+
     CONTEXT:SetExtraHeight( btnH + btnBottomMargin - 20 )
 
     function CONTEXT:_addDefaultButtons()
         self:AddButton( "Yes", Color( 0, 255, 0 ), true )
         self:AddButton( "No", Color( 255, 0, 0 ), false )
     end
 
-    function CONTEXT:AddButton( text, col, ... )
+    function CONTEXT:AddButtonAligned( text, col, alignment, ... )
         col = col or Color( 255, 255, 255 )
+        alignment = alignment or CFCNotifications.ALIGN_CENTER
```

Comment:
I don't think alignment should be optional for a function with Aligned in the name, if they wanna use the default, they should use normal AddButton.

Reply:
I just left it as technically optional for in case they leave it out like a doofus, so it just moves along without triggering an error.

Reply:
But I think it *should* error if they leave out alignment, otherwise we're allowing them to use the wrong function for the job, which is bad practice. This function has a specific purpose, and thats to add a button with alignment.